docs/services/navigation/_index.md

```diff
@@ -135,15 +141,14 @@ Go to your robot's **Code Sample** tab on the [Viam app](https://app.viam.com) f
 
 ### Mode
 
-{{% alert title="Stability Notice" color="note" %}}
-
-The `manual` mode is experimental and will have no effect.
+Get the `Mode` the service is operating in.
 
-{{% /alert %}}
+There are three options for modes: `MODE_UNSPECIFIED`, `MODE_MANUAL`, or `MODE_WAYPOINT`.
 
-Get the `Mode` the service is operating in.
-There are two options for modes: `manual` and `waypoint`.
-If you set the mode to `waypoint`, the service starts looking for added waypoints, effectively beginning navigation.
+- `MODE_UNSPECIFIED`: Do not perform any mode of autonomous navigation.
```

Comment:
Question re: `any mode`: Is it possible to purposefully set `MODE_UNSPECIFIED` myself, or does it only appear for a newly-created service until otherwise configured? If the former, `MODE_UNSPECIFIED` technically becomes a `Mode` itself, for example: as returned by `setMode`. If the latter, than this language is correct.